src/components/left-nav.tsx

```diff
@@ -1,16 +1,54 @@
-import { observer } from "mobx-react";
+import { inject, observer } from "mobx-react";
 import * as React from "react";
 
+import { IAllStores } from "..";
```

Comment:
This is confusing at first glance -- another reason to separate the store(s).

Reply:
That was VSCode doing an automatic import.